### PR TITLE
Pass the wrapper object to the validation invokeMethods

### PR DESCRIPTION
**Pull request by snirs90**
Passing the entity wrapper so we can validate fields with other fields like matching password, etc..
- Still missing tests

### CODE REVIEW

File: includes/EntityValidateBase.php [hunk 1]
```diff
@@ -172,12 +172,14 @@ public function validate($entity, $silent = FALSE) {
    *  Determine if we need to assign the from the callback to the field. Default
    *  to FALSE.
    */
-  protected function invokeMethods(EntityMetadataWrapper $property_wrapper, array $methods, $assign_value = FALSE) {
+  protected function invokeMethods(EntityMetadataWrapper $wrapper, $property_name, array $methods, $assign_value = FALSE) {
```
Comment (amitaibu) [inline includes/EntityValidateBase.php:177]: fix `@param`

Comment (snirs90) [inline includes/EntityValidateBase.php:177]: Fixed

**Comment by amitaibu** [comment]
Tests are failing, please check.

**Comment by snirs90** [comment]
Fixed test.

**Comment by RoySegall** [comment]
This is already been merged. Closing.

**Pull request closed without merging**